.gitlab-ci.yml

```diff
@@ -112,6 +112,7 @@ fossa:
     variables:
       - $CI_COMMIT_BRANCH == "main"
       - $CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+.*/
+      - $CI_COMMIT_BRANCH == "upgrade-debian-docker-instrumentation"
```

Comment:
remove this condition?

Reply:
Sure. this is temporary. I've been trying to trigger gitlab CI